### PR TITLE
WIP: Added helm3 option to chart_app

### LINKED ISSUE #20: Add --helm3 flag to chart app
## Description

Add --helm3 flag to chart app

The `arkade install chart` command should have helm3 support and use it by default.

### PR DESCRIPTION
**Pull request by kadern0**
Signed-off-by: kadern0 <kaderno@gmail.com>

Fixes #20

### CODE REVIEW

**Comment by aidun** [comment]
The code looks okay, without running it. I will check it tomorrow. I think you have a problem with go.mod there is a dependency to k8s which is not needed, you should remove it. Maybe you have also an issue with go fmt, Travis complains about something.

File: cmd/apps/chart_app.go [hunk 1]
```diff
@@ -131,12 +136,25 @@ before using the generic helm chart installer command.`,
...
+			}
 
+		} else {
+			outputPath = path.Join(chartPath, "chart/rendered")
+			err = templateChart(chartPath, chartName, namespace, outputPath, "values.yaml", setMap)
+			if err != nil {
+				return err
+			}
+		}
 		err = kubectl("apply", "--namespace", namespace, "-R", "-f", outputPath)
```
Comment (aidun) [inline cmd/apps/chart_app.go]: This part should be only for helm2. The helm 3 upgrade don't need it and is causing the strange error.

File: go.mod [hunk 2]
```diff
@@ -8,4 +8,5 @@ require (
 	github.com/sethvargo/go-password v0.1.2
 	github.com/spf13/cobra v0.0.5
 	golang.org/x/crypto v0.0.0-20190701094942-4def268fd1a4
+	k8s.io/apimachinery v0.17.3
```
Comment (aidun) [inline go.mod]: This dependency should be removed. `go mod tidy` can fix it.

File: cmd/apps/chart_app.go (hunk 1 above)
Comment (aidun) [reply cmd/apps/chart_app.go]: the `kubectl apply -f` is meant here.

File: go.mod (hunk 2 above)
Comment (kadern0) [reply go.mod]: Thank you for your help, it seems to be working fine now. I've submitted a new PR. 

I cannot @ you for some reason.

**Pull request closed without merging**